CHANGES.md

```diff
@@ -12,7 +14,7 @@
 - **Breaking**: Removed `Rasterband::read_as_array`, changed signature of `Rasterband::read_block` to return a `Buffer<T>`.
 - **Breaking**: `Rasterband::write` and `Rasterband::write_block` now require a `&mut Buffer<T>` to handle possible case of drivers temporarily mutating input buffer.
 
-   - <https://github.com/georust/gdal/pull/494>
+  - <https://github.com/georust/gdal/pull/494>
```

Comment:
I swear this happens every time :roll_eyes:.

Reply:
same